### DISCUSSION TITLE
Feedback / report button

### DISCUSSION THREAD

**Discussion by fcFn** [typehero/typehero#1266] [General]
Maybe the "Feedback" button should be renamed "Report" since its only purpose appears to be reporting abuse, not issues with or improvements for the challenge.

<details>
<summary>
Not really feedback
</summary>
Also please tell Frymagen that he could use Ctrl+C or remap some other key combination to switch back to normal mode 😁
</details>

**Comment by vegeta321311** [comment]
#AA00FF

**Reply by vegeta321311** [reply]
![profile-default.jpg](https://github.com/typehero/typehero/assets/144423495/5e766a44-86b6-4fee-b2c3-79057486dbf0)
